### PR TITLE
Changing logo and fixing form snag list

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- Updates the paultons logo to their new design (removes words shortbreaks)
- re-sizes SVGs in buttons to base font height
- Removes padding on buttons in grouped inputs
- adds focus and border to input boxes
- changes the colour of grouped input buttons
- Add styles for shortbreaks platform search form

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
The styles for shortbreaks platform search form need to eventually be removed from brands as they are very app specific, this work is temporary until i get a PR into the search form to change the code.

#### Screenshots / Screencast
screen shot 2018-01-10 at 11 46 07

#### What gif best describes how you feel about this work?
giphy

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/paultons/src/less/variables.less [hunk 1]
```diff
@@ -210,5 +212,5 @@
 //
 //##
 
-@logo-width:151px;
-@logo-height:121px;
+@logo-width:150px;
+@logo-height:140px;
```
Comment (jlsorak) [inline brands/paultons/src/less/variables.less]: The logo feels very small on desktop, I appreciate it needs to be small on mobile but what's your opinion on making it larger on desktop screens? 

(The screenshots are of the old logo but the same applies)

screen shot 2018-01-10 at 11 51 31

screen shot 2018-01-10 at 11 51 47

File: brands/paultons/src/less/buttons.less [hunk 2]
```diff
@@ -42,3 +46,20 @@
...
+
+.btn svg {
+  width:1em;
+}
+
+// This is app specific and will be reviewed for removal.
+.searchform-partycomposition .btn-info {
+  text-align:left;
+  .form-control;
+}
```
Comment (jlsorak) [inline brands/paultons/src/less/buttons.less]: Would it more suitable for me to put these straight in to the-wall instead of brands?

Comment (jodiedoubleday) [reply brands/paultons/src/less/buttons.less]: yea but i think you will need to put it in the wall and ssg (as its the search form)

File: brands/paultons/src/less/variables.less (hunk 1 above)
Comment (jodiedoubleday) [reply brands/paultons/src/less/variables.less]: im happy to make larger on bigger screens, i think brands already has some code to do this, ill investigate as a separate jira

File: brands/paultons/src/less/buttons.less (hunk 2 above)
Comment (jlsorak) [reply brands/paultons/src/less/buttons.less]: Fair point, happy to leave this here for now due to time constraints. Could be something we address as part of the washup.

**Pull request merged**